spec/rubocop/cop/style/method_called_on_do_end_block_spec.rb

```diff
@@ -5,62 +5,57 @@
 
   context 'with a multi-line do..end block' do
     it 'registers an offense for a chained call' do
-      inspect_source(cop, <<-END.strip_indent)
+      expect_offense(<<-RUBY.strip_indent)
         a do
           b
         end.c
-      END
-      expect(cop.offenses.size).to eq(1)
-      expect(cop.highlights).to eq(['end.c'])
+        ^^^^^ Avoid chaining a method call on a do...end block.
+      RUBY
     end
 
     it 'accepts it if there is no chained call' do
-      inspect_source(cop, <<-END.strip_indent)
+      expect_no_offenses(<<-RUBY)
```

Comment:
The extra whitespace messes up with the positions (and the way the examples are printed), that's why we strip it everywhere.